### ISSUE TITLE
Why `op2.cbar_force[i]` does not have the same format as `op2.cquad4_force[i]`?

### ISSUE THREAD

**Issue by saullocastro** [SteveDoyle2/pyNastran#221]
Dear Steve,

I've been working with the `pyNastran.op2.op2.OP2` class recently, reading the op2 with `vectorized=True` and noticed that when I do:

    op2 = OP2()
    op2.read_op2('model.op2', vectorized=True)
    op2.cquad4_force[1].element

I get an array containing the elements corresponding to the second dimention of the array `op2.cquad4_force[1].data`.

The problem is, when I querry the results from:

    op2.cbar_force[1]

I don't have the attribute `element` and it can be noticed that the outputs are stored in a different way, since:

    op2.cbar_force[1].axial

gives a dictionary with the output of each element where the axial force was requested.

Do you have plans to make all outputs vectorized? Would you like some help on that?

**Comment by SteveDoyle2** [comment]
The plan is to eventually make everything vectorized.  I was focusing more on the stress/strain results first, so forces are a lower priority.  I would love the help though.

I've tried to make classes follow the "Type" + "Name" + "Vectorized" style, so Real/Complex, Displacement/Stress/Strain/etc., and Array (if it's vectorized).  Array makes it a bit clearer, so RealLoadVector is the non-vectorized version and RealLoadVectorArray is the vectorized version.

I added the RealBarForceArray class, but there's a corresponding RealBar100ForceArray class that needs to be added for bars with 3-10 stations.  Similarly, there is a RealCBeamForceArray and a RealPlateBilinearForceArray class that need to be written.  There are a lot of other element types that are unvectorized (many of which don't have write_f06 methods yet).

If you have any questions, let me know.
